addons/website/models/ir_ui_view.py

```diff
@@ -153,7 +153,9 @@ def render(self, values=None, engine='ir.qweb'):
 
             values = qcontext
 
-        return super(View, self.with_context(new_context)).render(values, engine=engine)
+        if self._context != new_context:
```

Comment:
forget this, self.with_context preserves the prefetching... So, are you sure this line makes a significant difference?